core/templates/pages/certificate-assessment-player-page/certificate-assessment-conversation-skin.component.html

```
@@ -1,20 +1,102 @@
+        [type]="getQuestionInputType()"
+        class="certificate-assessment-free-response-input"
+        [placeholder]="currentQuestion.placeholder || ''"
+        [value]="freeResponse"
+        #freeResponseInput
+        (input)="updateFreeResponse(freeResponseInput.value)">
+    </div>
+  </div>
+
+  <div class="certificate-assessment-actions">
+    <button
+      *ngIf="currentQuestionIndex > 0"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)